pkgs/by-name/rt/rtcqs/package.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, fetchPypi
+, python3
+, tk
+}:
+
+let
+  pname = "rtcqs";
+  version = "0.6.2";
+in
+python3.pkgs.buildPythonApplication {
+  inherit pname version;
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-DfeV9kGhdMf6hZ1iNJ0L3HUn7m8c1gRK5cjtJNUAvJI=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [ setuptools ];
+  propagatedBuildInputs = [ tk ] ++ (with python3.pkgs; [ pysimplegui ]);
```

```suggestion
  dependencies = with python3.pkgs; [ tkinter pysimplegui ]);
```

(I'm not very sure about `python3Packages.tkinter`)

---

is it okay to use overlays/overrides in nixpkgs? **pysimplegui** is marked as broken and version 5 is now unfree???
checking upstream pyproject.toml of **rtcqs**, it only needs:

```
dependencies = [
    "PySimpleGUI == 4.60.5",
]
```